apps/desktop/src/lib/components/ProjectSetupTarget.svelte

```diff
@@ -248,7 +258,7 @@
 		</SetupFeature>
 	</div>
 	<div class="action-buttons">
-		<BackButton>Cancel</BackButton>
+		<BackButton preMouseDown={deleteProject}>Cancel</BackButton>
```

Comment:
Do you think we could just introduce a `<Button>Cancel</Button>` here instead of using the `<BackButton>`? That way we don't need to introduce the `preMouseDown` handler.

Reply:
I'm not entirely sure what you mean...

Do you mean to use the generic `Button` and set `onmousedown` to delete the project and add the existing implementation of `BackButton`'s `onmousedown`?
Or do you mean a to create a `CancelButton` with the delete project behavior?

Reply:
Yeah sorry, I mean a generic button with `onmousedown` to delete the project!

Reply:
I replaced `BackButton` with `Button`. Is the inline-ing still relevant?